FRONT/src/app/service/kanban.service.ts

```diff
@@ -0,0 +1,84 @@
+import { HttpClient, HttpHeaders } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { Router } from "@angular/router";
+import { Subject } from "rxjs";
+import { AuthGuardService } from "../guards/auth-guard";
+import { Card } from "../model/card.model";
+
+@Injectable({
+  providedIn: "root",
+})
+export class KanbanService {
+  private baseUrl = "http://0.0.0.0:5000";
+
+  cards!: Card[];
+
+  isAuth: boolean = false;
+  token!: any;
+
+  cardsChange = new Subject();
+
+  constructor(private http: HttpClient) {}
+
+  sendAuthRequest(login: string, senha: string) {
+    return this.http.post(this.baseUrl + "/login", {
+      login: login,
+      senha: senha,
+    });
+  }
+
+  saveToken(data: any) {
+    this.isAuth = true;
+    localStorage.setItem("token", data);
+    this.token = data;
+  }
+  isUserAuth() {
+    return this.isAuth;
+  }
+
+  getCards() {
+    const headers = new HttpHeaders()
+      .set("Content-Type", "application/json")
+      .set("Authorization", "Bearer " + localStorage.getItem("token"));
+
+    return this.http.get<Card[]>(this.baseUrl + "/cards", { headers: headers });
+  }
+
+  postCards(titulo: string, conteudo: string) {
+    let card = new Card("", titulo, conteudo, "todo");
+    const headers = new HttpHeaders()
+      .set("Content-Type", "application/json")
+      .set("Authorization", "Bearer " + localStorage.getItem("token"));
+
+    return this.http.post<Card[]>(this.baseUrl + "/cards/", card, {
```

Comment:
O que deve ficar entre <> nas requisições http, é o formato de dado que se espera como resposta da requisição. Conforme a documentação, o que se espera como resposta do DELETE é um array de cards, portanto, o correto seria `this.http.delete<Card[]>`. No caso do PUT, seria `this.http.put<Card>`. No caso do POST, o correto seria `this.http.post<Card>`.